sql/driver/wire.proto

```diff
@@ -48,6 +48,7 @@ message Datum {
     bytes bytes_val = 4;
     string string_val = 5;
     Timestamp time_val = 6;
+    int64 interval_val = 7;
```

Comment:
cool. I suppose we can also add a
int64 date_val in the future, or in this change if you are so inclined.